profile_queries.sh

```diff
@@ -127,6 +173,20 @@ generate_random_schema() {
     done
 }
 
+# Function to check whether the query has the word 'PROFILE' in it
+has_profile() {
+    # Execute the query and capture the output
+    OUTPUT=$(vsql -c "SELECT request FROM query_requests WHERE transaction_id=$TXN_ID AND statement_id=$STMT_ID AND request ILIKE 'PROFILE%';")
+
+    # Check if the output contains more than just the header and row count
+    if echo "$OUTPUT" | grep -qE '^[[:space:]]*PROFILE'; then
```

Comment:
Make the grep switches -qEi so that you pick up PROFILE case-independently. You are allowing for that in the vsql command above but not allowing it here.